fmt/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "forge-fmt"
```

Comment:
I'm thinking, should we name it `solang-fmt` instead? Or do we just copy it over from time to time leaving as much as possible as it is?

Reply:
It's a pretty useful crate TBH. We might want to publish it in the future.

So I agree, why not rename it to `solang-fmt`.

Reply:
I've changed my mind. Let's just vendor it for now.